api/annotations/index.js

```diff
@@ -0,0 +1,5 @@
+const annotationRouter = require('./annotation');
```

Comment:
you don't need this tbh
please look at
the 
`api/service/` folder we've created
https://github.com/hmcts/rpa-jui-webapp/tree/master/api/services

Reply:
are you on holiday?

Reply:
maybe...